### PR TITLE
Refactored to Follow Best Practices

### PR DESCRIPTION
**Pull request by candacejwelch**
Using ReSharper and our Knowledge of best practices, we've improved upon this project.

### CODE REVIEW

File: RefactorKata/RefactorKata/Program.cs [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
+                Console.WriteLine("This product is called: " + product.Name);
             }
-            Conn.Dispose();
-            Console.WriteLine("Products Loaded!");
-            for (int i =0; i< products.Count; i++)
+        }
+
+        private static IEnumerable<Product> GetProducts()
+        {
+            using (var conn =
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: Let's extract the connection string to the App.config

File: RefactorKata/RefactorKata/Program.cs [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
-                prod.name = reader["Name"].ToString();
-                products.Add(prod);
+                Console.WriteLine("This product is called: " + product.Name);
             }
-            Conn.Dispose();
-            Console.WriteLine("Products Loaded!");
-            for (int i =0; i< products.Count; i++)
+        }
+
+        private static IEnumerable<Product> GetProducts()
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: Let's comment document this method

File: RefactorKata/RefactorKata/Program.cs (hunk 1 above)
Comment (candacejwelch) [reply RefactorKata/RefactorKata/Program.cs]: No App.config, I (Cody) believe it's a .CORE not .NET project. Please advise.

Comment (soitgoes) [reply RefactorKata/RefactorKata/Program.cs]: We'll review tomorrow.  I have a refactor exercise that will correct

**Pull request closed without merging**